contrib/babelfishpg_common/sql/upgrades/spatial_types--3.3.0--3.4.0.sql

```diff
@@ -1,3 +1,1300 @@
 -------------------------------------------------------
 ---- Include changes related to spatial types here ----
 -------------------------------------------------------
+
+CREATE OR REPLACE FUNCTION sys.geometryin(cstring)
+    RETURNS sys.GEOMETRY
+    AS '$libdir/postgis-3', 'LWGEOM_in'
+    LANGUAGE 'c' IMMUTABLE STRICT PARALLEL SAFE;
+
+CREATE OR REPLACE FUNCTION sys.geometryout(sys.GEOMETRY)
+	RETURNS cstring
+	AS '$libdir/postgis-3','LWGEOM_out'
+	LANGUAGE 'c' IMMUTABLE STRICT PARALLEL SAFE;
+
+CREATE OR REPLACE FUNCTION sys.geometrytypmodin(cstring[])
+	RETURNS integer
+	AS '$libdir/postgis-3','geometry_typmod_in'
+	LANGUAGE 'c' IMMUTABLE STRICT PARALLEL SAFE;
+
+CREATE OR REPLACE FUNCTION sys.geometrytypmodout(integer)
+	RETURNS cstring
+	AS '$libdir/postgis-3','postgis_typmod_out'
+	LANGUAGE 'c' IMMUTABLE STRICT PARALLEL SAFE;
+
+CREATE OR REPLACE FUNCTION sys.geometryanalyze(internal)
+	RETURNS bool
+	AS '$libdir/postgis-3', 'gserialized_analyze_nd'
+	LANGUAGE 'c' VOLATILE STRICT;
+
+CREATE OR REPLACE FUNCTION sys.geometryrecv(internal)
+	RETURNS sys.GEOMETRY
+	AS '$libdir/postgis-3','LWGEOM_recv'
+	LANGUAGE 'c' IMMUTABLE STRICT PARALLEL SAFE;
+
+CREATE OR REPLACE FUNCTION sys.geometrysend(sys.GEOMETRY)
+	RETURNS bytea
+	AS '$libdir/postgis-3','LWGEOM_send'
+	LANGUAGE 'c' IMMUTABLE STRICT PARALLEL SAFE;
+
+DO $$
+DECLARE
+    exception_message text;
+BEGIN
+    CREATE TYPE sys.GEOMETRY (
+        INTERNALLENGTH = variable,
+        INPUT = sys.geometryin,
+        OUTPUT = sys.geometryout,
+        SEND = sys.geometrysend,
+        RECEIVE = sys.geometryrecv,
+        TYPMOD_IN = sys.geometrytypmodin,
+        TYPMOD_OUT = sys.geometrytypmodout,
+        DELIMITER = ':',
+        ALIGNMENT = double,
+        ANALYZE = sys.geometryanalyze,
+        STORAGE = main
+    );
+
+EXCEPTION WHEN OTHERS THEN
```

Comment:
Issue: We should not ignore all the errors, rather just ignore `duplicate_object` error for which we will throw warning saying that the type already exists but we will fail for all other errors.

Reply:
Done